src/Core/Services/Implementations/AzureQueueEventWriteService.cs

```diff
@@ -4,12 +4,15 @@
 using Newtonsoft.Json;
 using Bit.Core.Models.Data;
 using Bit.Core.Settings;
+using System.Linq;
+using System.Text;
 
 namespace Bit.Core.Services
 {
     public class AzureQueueEventWriteService : IEventWriteService
     {
         private readonly QueueClient _queueClient;
+        private const int MAX_MESSAGE_BODY = 128000;
```

Comment:
published value for this message size is only 64kb, but the error response is >130000. perhaps we should limit to the published value?

Reply:
Yes, published value is best... they are configurable, however better to keep these capped to standard size.